### ISSUE TITLE
[Quest] Tome of Valor (alliance paladin quest)

### ISSUE THREAD

**Issue by Filip-Piekarek** [dalaranwow/dalaran-wow#5867]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
After failing Tome of Valor quest and if it is then abandoned, I cannot get the quest again from Duthorian Rall.
**Expected behaviour**: Tell us what should happen instead
Quest should be available from Dutharian Rall
**Steps to reproduce the problem**:

1. Accept Tome of Valor quest
2. Talk to Daphne Stilwell, die trying to defend her, and thus she dies too
3. She respawns, cannot talk to her again
4. Abandon quest
5. Go back to Dutharian Rall, quest is not available (he has a blue question mark above his head, but only has a bit of dialogue text). Tome of Valor quest is not available.

**Include proofs for this behaviour**
![Capture](https://user-images.githubusercontent.com/46203053/78098816-14482e80-73ae-11ea-9ae8-da5207139f8c.PNG)

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**
